### ISSUE TITLE
White space in featured speaker section

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-orga-server#3297]
The featured speaker section has a white space issue.

![screenshot from 2017-03-04 16-33-32](https://cloud.githubusercontent.com/assets/1583873/23577372/7a7df792-00f8-11e7-86ba-ebc708779e10.png)

**Comment by poush** [comment]
I'm on it

**Comment by poush** [comment]
@niranjan94 I think the issue is related with the html unclosed html tags from wysiwyg editor  for entering speaker's bio. Since we are generating substring of 120 characters length of speaker's bio. Can we remove all HTML tags from bio to make sure this not happens ?

**Comment by niranjan94** [comment]
@poush i don't think we have any unclosed tags. We're sanitizing the html before saving it. This  is probably a bootstrap grid related issue. Please check. 

Removing something without a valid reason just as a hack to fix a bug is not an option.

**Comment by poush** [comment]
@niranjan94 I think I wasn't clear here. Firstly No, I'm not saying about the content we are receiving from the editor. The issue is that we are generating substring of speaker's bio to limit it to 120 characters. Now if the content in speaker bio was something like this

```
<p style="font-weight: 500">
some 100 lines here
</p>
```

Now if we make substring of it, we will end up with something like 

```
<p style="font-weight: 500">
some 100 more chars 
```
This will cause ```p``` tag to be left open.
Now we can also escape html tags but it will show html tags on the bio. Also due to this html also counts in speaker's bio. 

I am proposing two solutions for this, First one is best which is to apply the limit of 120 chars in bio input field and Second one is to remove all html tags from bio

**Comment by SaptakS** [comment]
@mariobehling @niranjan94 I think this issue has been fixed  by #3294. Please check. @poush we are no longer taking a substring of text, so your query is also answered I think.
